src/diffusers/models/resnet.py

```diff
@@ -640,7 +640,200 @@ def forward(self, input_tensor, temb):
         return output_tensor
 
 
-# unet_rl.py
+class Upsample3D(nn.Module):
+    def __init__(self, channels, use_conv=False, out_channels=None):
+        super().__init__()
+        self.channels = channels
+        self.out_channels = out_channels or channels
+        self.use_conv = use_conv
```

Comment:
If I'm reading teh code in `unet_3d_blocks.py` correctly `use_conv` is always set to `True`. Therefore let's just remove this input argument if it's never changed. 

This PR adds a lot of code, we should try to simplify every new block and every new feature to a maximum. Code that is **not** needed by the provided checkpoints **should be deleted**. 

cc @sayakpaul would be great if you could also check here